app/services/proofing/socure/id_plus/request.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Socure
+    module IdPlus
+      class RequestError < StandardError
+        def initialize(wrapped)
```

Comment:
it wasn't clear to me just from the name `wrapped` what this was wrapping, some ideas:
1. call it `error` or `wrapped_error`
2. type annotation?
```suggestion
        # @param [StandardError] wrapped
        def initialize(wrapped)
```